jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -53,5 +53,10 @@
        <span style='padding-left: 4px' v-if="results_available">{{number_of_results}}</span>
     <v-row>
 
+    <!-- Adding the div wrapper here for the jupyter-widget -->
+    <div class="jupyter-widget-container">
```

Comment:
what does this actually do?  Everywhere else we have the table, I don't think it needs to be wrapped by anything.

Reply:
I removed it and it doesn't make any noticeable changes. I will remove the wrapper.